CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added a new entity named `predictionResponse` to reflect the predictions which is data that can be used to anticipate and plan for future consumption demand . 
+- Added a new entity named  `Predictions` to reflect the prediction endpoint paylaod structure .   
+- Added a new method named `LicenseClient::getPredictions()` in order to get Predictions for API Endpoint .  
+
+## [0.10.0] - 2023-02-09
```

Comment:
This should not show up as new code. You have a rebase problem. You should not have 2 commits on your PR